src/main/java/io/dockstore/provision/DOSPluginUtil.java

```diff
@@ -31,6 +31,8 @@
     private static final int DG_PREFIX = 1;
     private static final int DG_UUID = 2;
 
+    private HttpURLConnection conn;
```

Comment:
It would be better if this were a method variable rather than a field, IMHO.

I guess it's tricky to close the connection though.